test/abel.lean

```diff
@@ -7,6 +7,7 @@ example [add_comm_group α] (x : α) : x - 0 = x := by abel
 example [add_comm_monoid α] (x : α) : (3 : ℕ) • a = a + (2 : ℕ) • a := by abel
 example [add_comm_group α] : (3 : ℤ) • a = a + (2 : ℤ) • a := by abel
 example [add_comm_group α] (a b : α) : a-2•b = a -2•b := by abel
+example [add_comm_group α] {a : α} : 0 + a = a := by abel
```

Comment:
```
example [add_comm_group α] {a : α} : 0 + a = a := by abel1
```
(The bug only turned up for `abel1`, or for somewhat more elaborate expressions using `abel`)